### ISSUE TITLE
Pulse progress bar never stops in v0.16

### ISSUE THREAD

**Issue by yaroslav-shabalin** [glasswalk3r/siebel-monitoring-tools#4]
With transition from **Term::Pulse** to **Siebel::Srvrmgr::Exporter::TermPulse** in version 0.16 now there is an issue while running export_comps. After getting component definitions program doesn't quit and progress bar continues to move, timer increases. Program is not responsive to pressing Ctrl+C. Environment is Windows 7 x64, Starwberry Perl version 5.20.2. Rolling back to **Term::Pulse** resolves the problem. Below is sample console output:
```
C:\pl>export_comps -g *** -e *** -s *** -u *** -p *** -b c:\Siebel\8.1\Client_2\BIN\srvrmgr.exe -r "SAP"
found definitions for component SAPWfProcBatchMgr[            -   ] (16.675238 sec elapsed)
------------ GENERATING OUTPUT -------------a......[      /         ] (26.876356 sec elapsed)

create component definition SAPWfProcBatchMgr for component type BusSvcMgr component group Workflow run mode Batch full name "SAP Workflow Process Batch Manager" description "" with parameter MaxTasks=20,Method=RunBatch,ServiceName=Workflow Process Manager;

Connecting to Siebel and getting initial data......[           -    ] (102.300813 sec elapsed)
```

**Comment by glasswalk3r** [comment]
Bug acknowledged. Siebel::Srvrmgr::Exporter::TermPulse removed ```SUB{__die__}``` handler and probably that's why the child process is not killed with CRTL+C. Didn't check it although.
Unix-like process handling in Microsoft Windows does not work as expected in most cases, I'll probably rewrite my version of Term::Pulse to use ithreads to handle the animation.
For now, I recommend disabling the "pulse" animation by using the command line option "-q" of export_comps.pl.

**Comment by glasswalk3r** [comment]
Fixed on Siebel::Srvrmgr 0.17 release.
